### PR TITLE
Memtarget capacity

### PR DESCRIPTION
**Pull request by lplewa**
~~depends on https://github.com/oneapi-src/unified-memory-framework/pull/631~~

### CODE REVIEW

File: include/umf/memtarget.h [hunk 1]
```diff
@@ -31,6 +32,13 @@ typedef enum umf_memtarget_type_t {
 umf_result_t umfMemtargetGetType(umf_const_memtarget_handle_t hMemtarget,
                                  umf_memtarget_type_t *type);
 
+/// \brief Get capacity of the memory target.
```
Comment (lukaszstolarczuk) [inline include/umf/memtarget.h:35]: capacity? you mean the size of memory? in bytes?

File: test/memspaces/memtarget.cpp [hunk 2]
```diff
@@ -13,17 +14,39 @@ using umf_test::test;
...
         auto hTarget = umfMemspaceMemtargetGet(memspace, i);
         ASSERT_NE(hTarget, nullptr);
-        umf_memtarget_type_t type;
         auto ret = umfMemtargetGetType(hTarget, &type);
         EXPECT_EQ(ret, UMF_RESULT_SUCCESS);
         EXPECT_EQ(type, UMF_MEMTARGET_TYPE_NUMA);
     }
 }
 
+TEST_F(numaNodesTest, getCapacity) {
```
Comment (lukaszstolarczuk) [inline test/memspaces/memtarget.cpp:27]: do we have a negative test for it?

Comment (lplewa) [reply test/memspaces/memtarget.cpp:27]: done

File: include/umf/memtarget.h (hunk 1 above)
Comment (lplewa) [reply include/umf/memtarget.h]: Done

File: include/umf/memtarget.h [hunk 3]
```diff
@@ -31,6 +32,13 @@ typedef enum umf_memtarget_type_t {
...
+/// \param HMemtarget handle to the memory target
```
[the remark this answers is not in the document]
Comment (lplewa) [reply include/umf/memtarget.h]: Done

**Review by lukaszstolarczuk — APPROVED** [review]

**Pull request merged**